gammapy/scripts/tests/test_image_pipe.py

```diff
@@ -65,8 +65,8 @@ def test_image_pipe(tmpdir):
                          obs_table=data_store.obs_table, exclusion_mask=exclusion_mask)
     mosaic.make_images(make_background_image=True, for_integral_flux=True, radius=10.)
     # TODO: this fails on some machines, for unknown reasons.
-    #assert_equal(mosaic.maps['counts'].data.sum(), 2334.0)
-    #assert_almost_equal(mosaic.maps['bkg'].data.sum(), 1987.2088283446637)
-    #assert_almost_equal(mosaic.maps['exposure'].data.sum(), 5.9292301086257267e17)
-    #assert_almost_equal(mosaic.maps['significance'].lookup(center), 33.699068269123039)
-    #assert_almost_equal(mosaic.maps['excess'].data.sum(), 346.79117165533626)
+    assert_equal(mosaic.maps['counts'].data.sum(), 2334.0)
```

Comment:
This assertion fails in some builds:
https://travis-ci.org/gammapy/gammapy/jobs/137307389#L1697

I guess this is float rounding errors of the position of a single event at the image edge.
This can happen, it's normal.
You just have to change to e.g.
```
assert_allclose(mosaic.maps['counts'].data.sum(), 2334, atol=3)
```
Please also remove the `TODO` comment above the other asserts from `assert_almost_equal` to `assert_allclose`, to be in line with our [assert convention](http://docs.gammapy.org/en/latest/development/howto.html#assert-convention) in Gammapy.